### DISCUSSION TITLE
C++ 17 support for Qt 6

### DISCUSSION THREAD

**Discussion by MartinDelille** [QtExcel/QXlsx#221] [General]
One little question: when building with Qt6, how do you handle c++ version ? You require c++ 11 here: https://github.com/QtExcel/QXlsx/blob/master/QXlsx/CMakeLists.txt#L14 but Qt 6 needs C++ 17.

I had to patch your CMakeLists.txt to make it work on the conan recipe: https://github.com/conan-io/conan-center-index/pull/11364/files

**Comment by MartinDelille** [comment]
I submitted a PR to fix this problem: https://github.com/QtExcel/QXlsx/pull/223

**Comment by j2doll** [comment]
Dear @MartinDelille 

Some people are using this code in Qt5 LTS. 

We may need to adjust the C++ version accordingly.

I plan to end support for C++5 in the future.

<img src="https://www.qt.io/hs-fs/hubfs/subscription%20timeline.png?width=1370&name=subscription%20timeline.png" />
